tests/integration/targets/azure_rm_access_token/tasks/main.yml

```
@@ -0,0 +1,13 @@
+- name: Get access token for graphql
+  azure.azcollection.azure_rm_access_token:
+    scopes:
+    - https://graph.microsoft.com/.default
```

Wrong indentation: expected 6 but found 4
```suggestion
      - https://graph.microsoft.com/.default
```